### ISSUE TITLE
Can't delete files

### ISSUE THREAD

**Issue by noor-muradi** [glotlabs/gdrive#45]
I am getting failed getting file bad request (404) while trying to delete file, the uploading data to Google Drive is working fine but the delete command is giving error.

following attached screenshot is the error I am getting and the scopes which is given to API.

**error**
![image](https://github.com/glotlabs/gdrive/assets/24589033/7f27ec88-da1a-4a33-a800-dcd628ee9589)

**API scopes given**
![image](https://github.com/glotlabs/gdrive/assets/24589033/18c2365f-7a49-4237-ab4d-13bc4551193c)

**Comment by Glich440** [comment]
have you checked the web interface to check if it actualy exists?

**Comment by noor-muradi** [comment]
> have you checked the web interface to check if it actualy exists?

Yes, it exists, as mentioned, if I give delete command using file ID it succeed, I am only facing 404 error if try to delete using file name instead.

**Comment by Glich440** [comment]
does this happen with any file or only these ones?
